src/internet-analyzer/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.1.0rc7
```

Comment:
Could you please tell the reason why the version is `0.1.0rc7`, but not `0.`.0rc6`?

Reply:
Honestly, I wanted to go up by two because I wanted to show the time period since the last time we touched this extension. Been two years.

If you are fine, i'd like to keep it. Doesn't matter so if not, thats fine too.

Reply:
adjusted